examples/python/wt5001.py

```diff
@@ -53,7 +54,7 @@ def printUsage(progname):
     elif cmd == 2:
         myMP3Player.pause()
     elif cmd == 3:
-        myMP3Player.next()
+        next(myMP3Player)
```

Comment:
I don't think this is correct :) next() tells the player to skip to the next track.